hs/stack.yaml

```diff
@@ -3,6 +3,8 @@ resolver: lts-19.7
 ghc-options:
   "$everything": -haddock
 
+system-ghc: true
```

Comment:
What does this do? Does it break my machine?

Reply:
Accidental commit. fixes stack bug that causes me grief

Reply:
:)